connector/src/test/scala/com/microsoft/kusto/spark/authentication/KustoAuthenticationTests.scala

```diff
@@ -0,0 +1,59 @@
+package com.microsoft.kusto.spark.authentication
+
+import org.junit.runner.RunWith
+import org.scalatest.FlatSpec
+import org.scalatest.junit.JUnitRunner
+
+@RunWith(classOf[JUnitRunner])
+class kustoAuthenticationTests extends FlatSpec {
+  "KeyVaultAppAuthentication Equals" should "Check equality and inequality between different KeyVaultAppAuthentication" in {
```

Comment:
definitely should be part of sdk. unrelated to spark.

Reply:
One day :)